graphs/ent/graph.py

```diff
@@ -311,8 +380,14 @@ def __parse_bin(self):
                 if type(self.bin) == lief.PE.Binary:
                     self.type = 'PE'
                     log.debug('Parsed with lief as: {}'.format(self.type))
+                elif type(self.bin) == lief.ELF.Binary:
+                    self.type = 'ELF'
+                    log.debug('Parsed with lief as: {}'.format(self.type))
+                elif self.bin is None:
+					self.bin = None
+					log.debug('Parsing with lief resulted in an unrecognized format')
                 else:
-                    self.bin = None
+                    self.type = '_unk_exec'
```

Comment:
This _should_ never hit as if lief cannot parse the file it will throw a _pylief.bad_file exception